src/buildApplication/buildApplication.ts

```diff
@@ -26,6 +26,8 @@ export async function buildApplication({
 		await buildVercelOutput();
 	}
 
+	await purgePrivateFiles();
```

Comment:
sorry for being picky, but could we use `delete` instead of `purge`, `purge` might be ok but just a bit more ambiguous/less straightforward than `delete`?

I'd also make more clear what this files are, something along the lines of `deleteNextTelemetryFiles`

if I read such name I'd immediately get the gist of it, if instead I read `purgePrivateFiles` I'd need to look at the code (or the tsdoc) to understand what it is about.

what do you think?